documentation/blog/2023-12-01-whats-new-in-svelte-december-2023.md

```diff
@@ -0,0 +1,94 @@
+---
+title: "What's new in Svelte: December 2023"
+description: 'Svelte 5.0 announced at Svelte Summit Fall 2023'
+author: Dani Sandoval
+authorURL: https://dreamindani.com
+---
+
+Svelte Summit Fall 2023 was epic! Featuring talks from across the ecosystem and the launch of the beta version of Svelte 5.0! Individual videos for each talk can be found in [this playlist](https://www.youtube.com/watch?v=RT0aemVpvVE&list=PL8bMgX1kyZTh_wu-CruPkxk6sMbXXluJK), but the whole talk can be found [here](https://www.youtube.com/watch?v=pTgIx-ucMsY) (timestamps below):
+- [0:00](https://www.youtube.com/watch?v=pTgIx-ucMsY&t=0s) Hype music!
+- [18:20](https://www.youtube.com/watch?v=pTgIx-ucMsY&t=1100s) Exploring Svelte DevTools - Ignatius B (4 minutes)
+- [25:27](https://www.youtube.com/watch?v=pTgIx-ucMsY&t=1527s) How does SvelteKit fare as a SPA Framework - Henry Lie (25 minutes)
+- [53:00](https://www.youtube.com/watch?v=pTgIx-ucMsY&t=3180s) enhanced:img - Ben McCann (6 minutes)
```

Comment:
I think this one deserves a callout in the paragraph above 😄

Reply:
too bad! ;)